### PR TITLE
qa: krbd_exclusive_option.sh: update for recent kernel changes

### PR DESCRIPTION
**Pull request by idryomov**

### CODE REVIEW

**Comment by idryomov** [comment]
jenkins test make check

File: qa/workunits/rbd/krbd_exclusive_option.sh [hunk 1]
```diff
@@ -84,79 +109,107 @@ sudo rbd unmap $DEV
...
 sudo rbd unmap $DEV
-wait $SUDO_PID
+assert_unlocked
+
+DEV=$(sudo rbd map $IMAGE_NAME)
+assert_locked $DEV
+dd if=/dev/urandom of=$DEV bs=4k count=10 oflag=direct
+{ sleep 10; blacklist_add $DEV; } &
+PID=$!
+expect_false dd if=/dev/urandom of=$DEV bs=4k count=200000 oflag=direct
```
Comment (dillaman) [inline qa/workunits/rbd/krbd_exclusive_option.sh]: This seems potentially racy. You would need to ensure that the OSD has the updated OSD map w/ the blacklist entry for this to actually fail.

Comment (idryomov) [reply qa/workunits/rbd/krbd_exclusive_option.sh]: While it is racy in theory, this test case is doing 200000 I/Os at QD=1 with messenger failure injection enabled.  Uninterrupted it runs for at least a few minutes, so there should be more than enough time for the blacklist entry to propagate.

A CentOS 7 run turned up a udev-related race in a different test case, I'm pushing a fix.

**Comment by idryomov** [comment]
http://pulpito.ceph.com/dis-2019-12-09_19:17:44-krbd-master-testing-basic-smithi/
http://pulpito.ceph.com/dis-2019-12-09_20:30:29-krbd-master-testing-basic-smithi/

**Pull request merged**